BeamlineStatusLogger/sources.py

```diff
@@ -194,3 +198,25 @@ def read(self):
             timestamp = datetime.now(self.localtz)
             value = None
         return Data(timestamp, value, metadata=metadata)
+
+
+class TINEAttributeSource:
+    def __init__(self, device_address, attr, metadata={}, tz=None):
+        if tine is None:
+            raise tine_import_err
```

Comment:
Could you please add similar lines for `tango`to the `__init__` method of `TangoDeviceAttributeSource`?